src/enclave/enclave_init.c

```diff
@@ -231,16 +227,21 @@ int __libc_init_enclave(int argc, char** argv)
     libc.user_tls_enabled =
         sgxlkl_enclave->mode == SW_DEBUG_MODE ? 1 : sgxlkl_enclave->fsgsbase;
 
+    SGXLKL_VERBOSE("calling init_sysconf()\n");
     init_sysconf(
         sgxlkl_enclave->sysconf_nproc_conf, sgxlkl_enclave->sysconf_nproc_onln);
+
+    SGXLKL_VERBOSE("calling init_clock_res()\n");
```

Comment:
This type of output is super verbose, maybe we want a new flag or a trace for this. Maybe someone could look into extracting stack traces on error?

Reply:
I think that the whole tracing needs to be rethought (we have #215 for this), but I don't want to do this as part of this PR. If we catch some of the initialisation segfaults, I can do another PR that removes some of these messages again.

Having stack traces would be wonderful, but getting these to work with symbols would require some work because of the custom way how we load them.

Reply:
Yeah, you're right, that should be part of the general tracing changes. 

I think for the kernel and below, `oe_print_backtrace()` should work, which may help with some of our CI debugging. But yeah, correctly loading symbols from application code is likely a lot of effort.

Reply:
I tried it a while back, and it didn't work even for libsgxlkl.so symbols.

Reply:
I see! I might have a few spare minutes over the weekend, maybe I'll take another look at it.